ansible_collections/arista/avd/docs/getting-started/avd-aap.md

```diff
@@ -0,0 +1,326 @@
+
+    ![Survey save](../_media/getting-started/aap-avd/survey-save.png)
+
+    The `lab_pass` variable or whichever name you would prefer can be used in your project by assigning the correct variable value to the `ansible_password` variable.
+
+    ```yaml
+    vars:
+      ansible_user: arista
+      ansible_password: "{{ lab_pass }}"
+      ansible_network_os: arista.eos.eos
+    ```
+
+=== "Surveys - Enable"
+
+    Click the `Surveys Disabled` radio button to ensure the survey is enabled.
+
+    ![Survey enable](../_media/getting-started/aap-avd/survey-enabled.png)
+
+### Running the Template with CVP
+
+Below is an example of the playbook we are leveraging to build and deploy our configurations with CVP. The following section will have an alternate version that can be deployed directly to EOS nodes.
+
+```yaml
+---
+- name: Manage Arista EOS EVPN/VXLAN Configuration
+  hosts: ATD_FABRIC
+  connection: local
+  gather_facts: false
+  collections:
+    - arista.avd
+  vars:
+    fabric_dir_name: "{{ fabric_name }}"
+  tasks:
+
+    - name: Generate intended variables
+      import_role:
+        name: eos_designs
+
+    - name: Generate device intended config and documentation
+      import_role:
+        name: eos_cli_config_gen
+
+- name: Configuration deployment with CVP
+  hosts: cv_servers
+  connection: local
+  gather_facts: false
+  collections:
+    - arista.avd
+  tasks:
+    - name: Provision CVP with AVD configuration
+      import_role:
+        name: eos_config_deploy_cvp
+      vars:
+        container_root: 'ATD_FABRIC'
+        configlets_prefix: 'AVD'
+        state: present
+        execute_tasks: true
+```
+
+We have everything we need to run our job template now.
+
+=== "Templates Run"
+
+    - On the left pane, click on `Templates`.
+    - Click on the `Launch Template` icon to run the job template.
+    - In the new pane, click `Next`.
+    - In the `Launch` pane, click `Launch`.
+
+    ![Run job](../_media/getting-started/aap-avd/run-job.png)
+    ![Launch job](../_media/getting-started/aap-avd/launch-job.png)
+
+=== "Jobs"
+
+    On the left pane, select `Jobs`. You may see a series of updates. For example, our source control is updating because our timeout is set to zero. The inventory has also been updating since we checked `Update on launch`. Last but not least, the job template will run.
+
+    ![Job updates](../_media/getting-started/aap-avd/job-updates.png)
+
+=== "Job - Output"
+
+    We can now click on the job run and see a successful execution.
+
+    ![Job output](../_media/getting-started/aap-avd/job-output.png)
+
+=== "CVP View"
+
+    From CVP's perspective, we can see a new container topology is created, and our change control workflow has been completed for us when leveraging the CVP Ansible collection.
+
+    ![CVP topology](../_media/getting-started/aap-avd/cvp-topo.png)
+    ![CVP Change Controls](../_media/getting-started/aap-avd/cvp-cc.png)
+
+### Running the Template directly to EOS nodes
```

Comment:
This sections feels out of place, since this playbook should already be in the example repo you inport, and you simply have to choose the right one in the template definition.

Reply:
I went back and forth on this and I agree, it would be fine to just remove. Thank you